src/main/preference.js

```diff
@@ -13,28 +13,51 @@ class Preference {
     this.cache = null
     this.staticPath = staticPath
     this.userDataPath = userDataPath
+    this.requiresUpdate = false
+
+    this.outdatedKeys = [
```

Comment:
I maybe make you confused.

I gave the example of `color`, `darkColor` and `lightColor`.

It's just an example, and we should compare the old preferences and new preferences in `userDataPath` and `staticPath`, and compute the outdate preferences and newly added preferences.

Then we delete the outdate keys and add the new ones.

Reply:
Ok, outdated keys are now removed.